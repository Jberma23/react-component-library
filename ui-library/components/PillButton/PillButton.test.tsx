import { render } from "@testing-library/react";
import { PillButton } from ".";

describe(PillButton, () => {
  it("renders with children and defaults", () => {
    const { asFragment } = render(<PillButton>Hot Dogs</PillButton>);
    expect(asFragment()).toMatchSnapshot();
  });
  it("passes all a11y checks", () => {
    expect(<PillButton>Kielbasa!</PillButton>).toBeA11yWhenRendered();
  });
});
