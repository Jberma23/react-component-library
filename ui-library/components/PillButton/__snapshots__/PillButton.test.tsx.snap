// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PillButton renders with children and defaults 1`] = `
<DocumentFragment>
  <button
    class="leading-body font-semibold text-body-regular w-fit px-[0.75rem] py-[0.125rem] border-1 rounded-[50px] whitespace-nowrap bg-primary-50 border-gray-700 text-gray-800 transition-colors duration-default hover:bg-primary-200 active:bg-primary-300"
  >
    Hot Dogs
  </button>
</DocumentFragment>
`;
